### PR TITLE
add documentation for alerts related to ceph osd 

### PR DESCRIPTION
**Pull request by weirdwiz**
this commit adds runbooks documentation for alerts related to ceph

### CODE REVIEW

**Review by agarwal-mudit — APPROVED** [review]
/lgtm

File: alerts/openshift-container-storage-operator/CephDataRecoveryTakingTooLong.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+### Recommended Actions
+
+1. **Check for Network Issues:**
+   Verify if the extended data recovery time is due to network issues by
+   following the steps in the provided Standard Operating Procedure (SOP) -
+   [Check Ceph Network Connectivity SOP](helpers/networkConnectivity.md).
+
+   If a network issue is identified, escalate to the ODF team by following the
+   steps [here](helpers/sre-to-engineering-escalation.md#procedure).
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/CephDataRecoveryTakingTooLong.md:34]: Where is this helper file?

File: alerts/openshift-container-storage-operator/CephDataRecoveryTakingTooLong.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+2. **Generic Debugging:**
+   Follow the general pod debug workflow outlined below to identify and
+   address potential issues.
+
+   - [Pod Debug Workflow](helpers/podDebug.md)
+   - [Gather Logs](helpers/gatherLogs.md)
+
+**Additional Resources:**
+
+- [Troubleshooting](helpers/troubleshootingCeph.md)
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/CephDataRecoveryTakingTooLong.md:45]: the file name is _troubleshootCeph.md_

File: alerts/openshift-container-storage-operator/CephOSDFlapping.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- [Pod Debug](helpers/podDebug.md)
+- [Gather Logs](helpers/gatherLogs.md)
+
+### Check if it is a Network Issue
+
+Check for network issues by following the steps in this
+[SOP](helpers/networkConnectivity.md). If it is determined to be a network
+issue, escalate to the ODF team using the steps outlined
+[here](helpers/sre-to-engineering-escalation.md#procedure).
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/CephOSDFlapping.md:37]: ... file missing

File: alerts/openshift-container-storage-operator/CephPgRepairTakingTooLong.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+placement groups, are taking longer than expected.
+
+## Diagnosis
+
+The alert is triggered when self-healing processes within Ceph are identified as
+taking too long. The suggested approach is to check for inconsistent placement
+groups and perform repairs using the provided Knowledgebase Article (KCS) -
+[KCS with PGRepair details](https://access.redhat.com/solutions/1589113).
+
+**Prerequisites:** [Prerequisites](helpers/prerequisites.md)
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/CephPgRepairTakingTooLong.md:23]: we do not have "prerequisites.md". I think not needed

File: alerts/openshift-container-storage-operator/CephPgRepairTakingTooLong.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+1. **Repair Placement Groups:** Execute the steps outlined in the KCS with
+   PGRepair details to identify and repair inconsistent placement groups.
+
+   [KCS with PGRepair details](https://access.redhat.com/solutions/1589113)
+
+2. **Check for Network Issues:** Verify if the extended repair time is due to
+   network issues by following the steps in the provided Standard Operating
+   Procedure (SOP) -
+   [Check Ceph Network Connectivity SOP](check-ceph-network-connectivity.md).
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/CephPgRepairTakingTooLong.md:37]: we do not have this file.. maybe you need "helpers/networkConnectivity"

File: alerts/openshift-container-storage-operator/CephPgRepairTakingTooLong.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+   If a network issue is identified, escalate to the ODF team by following the
+   steps [here](sre-to-engineering-escalation.md#procedure).
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/CephPgRepairTakingTooLong.md:40]: we do not have this file

File: alerts/openshift-container-storage-operator/CephPgRepairTakingTooLong.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+**Additional Resources:**
+
+- [Gather Logs](helpers/gather_logs.md)
+- [Troubleshooting](helpers/troubleshooting.md)
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/CephPgRepairTakingTooLong.md:45]: probably you really want to have ... "helpers/troubleshootCeph.md"

File: alerts/openshift-container-storage-operator/CephPgRepairTakingTooLong.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+   network issues by following the steps in the provided Standard Operating
+   Procedure (SOP) -
+   [Check Ceph Network Connectivity SOP](check-ceph-network-connectivity.md).
+
+   If a network issue is identified, escalate to the ODF team by following the
+   steps [here](sre-to-engineering-escalation.md#procedure).
+
+**Additional Resources:**
+
+- [Gather Logs](helpers/gather_logs.md)
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/CephPgRepairTakingTooLong.md:44]: gatherLogs.md

File: alerts/openshift-container-storage-operator/ODFPersistentVolumeMirrorStatus.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+```bash
+rbd mirror image status POOL_NAME/IMAGE_NAME
+```
+
+## Mitigation
+
+Review [RBD mirrorpod](helpers/podDebug.md) to find out more information about
+the problem.
+
+[gather_logs](helpers/gather_logs.md) to provide more information to support
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/ODFPersistentVolumeMirrorStatus.md:47]: gatherLogs.md

**Review by jmolmo — CHANGES_REQUESTED** [review]
review links to helpers files....

**Comment by weirdwiz** [comment]
updated the PR with the changes

**Review by jmolmo — APPROVED** [review]
LGTM

**Comment by weirdwiz** [comment]
ptal @umangachapagain

**Review by umangachapagain — APPROVED** [review]

**Pull request merged**